### ISSUE TITLE
[bug] v2: multiple webviews (mostly) white on load

### ISSUE THREAD

**Issue by kimhrac** [tauri-apps/tauri#10011]
### Describe the bug

I create a new window (using `tauri::WindowBuilder`) and adding 3 webviews to it (using `tauri::webview::WebviewBuilder`); one webview loads a local HTML file, and the other two loads Google search and Bing search.

The webviews are currently arranged so that they do not overlap (but they will later on). The following screenshot shows how it is supposed to be seen (at the top the local HTML) and the bottom part is divided 50/50 with Google search on the left and Bing on the right:
![tauri_multiple_webviews_both](https://github.com/tauri-apps/tauri/assets/30130752/38a5c082-56bd-490a-a855-8bfc847c82f3)

The problem is that most of the time the external websites (Google and Bing) are not shown at all...
![tauri_multiple_webviews_zero](https://github.com/tauri-apps/tauri/assets/30130752/937b3eb0-87ca-48a0-b45d-2011a108e5a1)
...or only one of them...
![tauri_multiple_webviews_single](https://github.com/tauri-apps/tauri/assets/30130752/f53fd967-38ad-41b1-b6f7-b4c839942340)

I'm not a Rust developer and Tauri is very new to me, so I don't know if it is a problem with my code, or Tauri (or one of its sub-components/-projects).

Since the problem occurs most of the time, but it sometimes works, I would conclude that it might be because the webviews are created more or less at the exact same time and something might conflict, but I'm not sure.

### Reproduction

My code looks like this:

```rust
pub fn run() {
    tauri::Builder::default()
        .setup(|app| {
            // Main window
            let window = tauri::WindowBuilder::new(app, "App")
                .title("Test of multiple Webviews")
                .inner_size(800., 600.)
                .build()
                .unwrap();

            // Add main webview
            let _webview_main = window.add_child(
                tauri::webview::WebviewBuilder::new("WebviewMain", WebviewUrl::App("index.html".into()))
                    .auto_resize(),
                LogicalPosition::new(0., 0.),
                LogicalSize::new(800., 200.),
            )?;

            // Add webview for site #1
            let _webview1 = window.add_child(
                tauri::webview::WebviewBuilder::new("Webview1", WebviewUrl::External(Url::parse("https://www.google.com/").expect("Failed to parse URL")), )
                    .auto_resize(),
                LogicalPosition::new(0., 200.),
                LogicalSize::new(400., 400.),
            )?;

            // Add webview for site #2
            let _webview2 = window.add_child(
                tauri::webview::WebviewBuilder::new("Webview2", WebviewUrl::External(Url::parse("https://www.bing.com/").expect("Failed to parse URL")), )
                    .auto_resize(),
                LogicalPosition::new(400., 200.),
                LogicalSize::new(400., 400.),
            )?;

            Ok(())
        })
        .run(tauri::generate_context!())
        .expect("error while running tauri application");
}
```

### Expected behavior

I would expect all three webviews to show content every time I launch my application.

### Full `tauri info` output

```text
[✔] Environment
    - OS: Windows 10.0.19045 X64
    ✔ WebView2: 124.0.2478.97
    ✔ MSVC: Visual Studio Community 2022
    ✔ rustc: 1.76.0 (07dca489a 2024-02-04)
    ✔ cargo: 1.76.0 (c84b36747 2024-01-18)
    ✔ rustup: 1.27.1 (54dd3d00f 2024-04-24)
    ✔ Rust toolchain: stable-x86_64-pc-windows-msvc (default)
    - node: 20.11.0
    - npm: 10.8.1

[-] Packages
    - tauri [RUST]: 2.0.0-beta.11
    - tauri-build [RUST]: 2.0.0-beta.9
    - wry [RUST]: 0.37.0
    - tao [RUST]: 0.26.1
    - @tauri-apps/api [NPM]: 2.0.0-beta.13
    - @tauri-apps/cli [NPM]: 2.0.0-beta.20

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../src
```

**Comment by kimhrac** [comment]
I've read somewhere that "by default multiple webview support is not enabled". Does this mean that I will have to enable unstable flag in order to have my application work? And if yes...

a) Is this something that will be worked on for an upcoming release?
b) What side-effects do I have to consider if I am going to enable unstable flag?

**Comment by FabianLars** [comment]
> Does this mean that I will have to enable unstable flag in order to have my application work? And if yes...

Yes.

> a) Is this something that will be worked on for an upcoming release?

Likely, yes. We don't know when though.

>  b) What side-effects do I have to consider if I am going to enable unstable flag?

Not only the API is unstable but the implementation/behavior too, make sure to double check everything. For example resizing tends to be a problem.

**Comment by kimhrac** [comment]
@FabianLars do you know enough to be able to tell where the problems related to this issue lies? I mean... is it in Tauri, WRY, TAO and/or somewhere else?

**Comment by FabianLars** [comment]
Not really, no. If you can't reproduce it in wry then the chance that it's tauri's fault is pretty high (but a fix may still require changes in wry). I don't think tao is at fault here but with how stupid OS apis are i wouldn't bet money on it (still the lowest target for me).

**Comment by finaIcutpro** [comment]
What is the flag that we have to enable? In my app, overlays are white and the entire window hangs..
